indexer/osm_editor.cpp

```diff
@@ -342,31 +342,37 @@ void Editor::DeleteFeature(FeatureType const & feature)
   Invalidate();
 }
 
-Editor::SaveResult Editor::SaveEditedFeature(FeatureType & editedFeature, string const & editedStreet,
-                                             string const & editedHouseNumber)
+namespace
 {
-  // Check house number for validity.
-  if (editedHouseNumber.empty() || feature::IsHouseNumber(editedHouseNumber))
-    editedFeature.SetHouseNumber(editedHouseNumber);
-  // TODO(AlexZ): Store edited house number as house name if feature::IsHouseNumber() returned false.
+constexpr uint32_t kStartIndexForCreatedFeatures = numeric_limits<uint32_t>::max() - 0xffff;
```

Comment:
We can’t use whole 4G range, because it is overlapped with already existing features.
I suppose 1M is pretty enough for the first release.